.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
```

Comment:
Similar to above, consider adding some detail as to the expected scope of a User Story.